config/01-ce-auth-defaults.conf

```diff
@@ -59,9 +59,11 @@ GSI_DAEMON_KEY  = /etc/grid-security/hostkey.pem
 GSS_ASSIST_GRIDMAP_CACHE_EXPIRATION = 30*$(MINUTE)
 
 # SSL settings
-AUTH_SSL_SERVER_CAFILE=/etc/pki/tls/certs/ca-bundle.crt
-AUTH_SSL_SERVER_CERTFILE=/etc/grid-security/condor/hostcert.pem
-AUTH_SSL_SERVER_KEYFILE=/etc/grid-security/condor/hostkey.pem
+# TODO: These are the new defaults in 8.9 so these settings can be
+# removed when we no longer need to support HTCondor < 8.9
+AUTH_SSL_SERVER_CAFILE = /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
Try wrapping these in
```
if version <= 8.8
...
endif
```
(version comparison here is unusual because `version <= 8.8` is true for 8.8.X for any X)

Reply:
I'm not sure if this buys us anything since we'll still want to remove this config when we stop supporting 8.9. An if block would make us sensitive to (unlikely) changes to the HTCondor default, too.